modules/show_map.R

```diff
@@ -149,7 +159,7 @@ show_map_server <- function(id, com_module) {
       data_snsrs <- get_locations()[[1]] %>% filter(., !grepl("KNMI|NL",station))
       
       # Update map with new markers to show selected
-      proxy <- leafletProxy('map') # set up proxy map
+      #proxy <- leafletProxy('map') # set up proxy map
```

Comment:
Hi @Hendricxw , Can this line have to do something why the map is updated every time a sensor is selected?

Reply:
I was trying this, just a minute ago. However, the map-refresh seems to be started by state_station and I do not know why (yet)...